### PR TITLE
Modified iface_namingmode testcases to get internal ports on voq system

### PR DESCRIPTION
**Pull request by oxygen980**
### Description of PR
Modified iface_namingmode testcases to get internal interfaces. 

Summary:
Fixes # (issue)
The script was missing internal interfaces and some testcases were failing.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Testcases were missing check on internal interfaces

#### How did you do it?
Modified testcases to get internal ports. 

#### How did you verify/test it?
verified on VOQ system.

### CODE REVIEW

File: tests/iface_namingmode/test_iface_namingmode.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def setup(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
 
     hwsku = duthost.facts['hwsku']
     minigraph_facts = duthost.get_extended_minigraph_facts(tbinfo)
-    port_alias_facts = duthost.port_alias(hwsku=hwsku, include_internal=False)['ansible_facts']
+    port_alias_facts = duthost.port_alias(hwsku=hwsku, include_internal=True)['ansible_facts']
```
Comment (ysmanman) [inline tests/iface_namingmode/test_iface_namingmode.py:40]: What internal ports will be included? Inband ports?  Also the change is not only for voq system, right?

**Review by ysmanman — APPROVED** [review]

File: tests/iface_namingmode/test_iface_namingmode.py [hunk 2]
```diff
@@ -336,8 +336,8 @@ def test_show_pfc_counters(setup, setup_config_mode):
     per the configured naming mode
     """
     dutHostGuest, mode, ifmode = setup_config_mode
-    pfc_rx = dutHostGuest.shell('SONIC_CLI_IFACE_MODE={} sudo show pfc counters | sed -n "/Port Rx/,/^$/p"'.format(ifmode))['stdout']
-    pfc_tx = dutHostGuest.shell('SONIC_CLI_IFACE_MODE={} sudo show pfc counters | sed -n "/Port Tx/,/^$/p"'.format(ifmode))['stdout']
+    pfc_rx = dutHostGuest.shell('SONIC_CLI_IFACE_MODE={} sudo show pfc counters -d all | sed -n "/Port Rx/,/^$/p"'.format(ifmode))['stdout']
+    pfc_tx = dutHostGuest.shell('SONIC_CLI_IFACE_MODE={} sudo show pfc counters -d all | sed -n "/Port Tx/,/^$/p"'.format(ifmode))['stdout']
```
Comment (wangxin) [inline tests/iface_namingmode/test_iface_namingmode.py:340]: Is this `-d all` option supported on release earlier than `202012`?

Comment (sanmalho-git) [reply tests/iface_namingmode/test_iface_namingmode.py:340]: '-d' option came in with multi-asic support. If 202012 supports multi-asic, then it should have the '-d' option

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 1 above)
Comment (arlakshm) [reply tests/iface_namingmode/test_iface_namingmode.py:40]: why do we need this change ? The port alias is not needed for inband ports I think we can ignore them.
The port alias is mainly needed for front panel ports.

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 2 above)
Comment (arlakshm) [reply tests/iface_namingmode/test_iface_namingmode.py:340]: why do we need this change ?

**Review by arlakshm — COMMENTED** [review]
as comments

**Pull request closed without merging**